ganttproject/src/main/java/biz/ganttproject/ganttview/TaskTable.kt

```diff
@@ -101,6 +102,17 @@ class TaskTable(
   val taskTableModel = TaskTableModel(taskManager, taskManager.customPropertyManager)
   private val task2treeItem = mutableMapOf<Task, TreeItem<Task>>()
 
+  lateinit var hiddenTaskCountObserver : (Int) -> Unit
+  private var hiddenTaskCount : Int by Delegates.observable(0) {
```

Comment:
I suggest moving this counter into TaskFilterManager, and using SimpleIntegerProperty from JavaFX which allows for adding change listeners in the runtime:

```
class TaskFilterManager {
   ...
   val hiddenTaskCount = SimpleIntegerProperty(0)
   ...
 }
 ```
 
 ```
class TaskTable {
   ...
   this.filters.hiddenTaskCount.set(filteredCount)
  ...
}

class GanttChartTabContentPanel {
  ...
  this.filterManager.hiddenTaskCount.addListener(..)
  ...
}
```

Also, we probably can replace the call to TaskTable::taskFilterActions with the direct access of TaskFilterManager properties.

Reply:
Done
Filter actions have been moved to filter manager